src/aere_parse.erl

```diff
@@ -37,17 +37,18 @@ commands() ->
     , {"load",
        {["l"], {min_args, 1}, "FILENAMES",
         [ "Loads files into the REPL. Adds the last file to the scope."
-        , "Cleans user variables and functions. Unloads previously loaded files."]
+        , "Cleans user variables, functions, types and contracts."
+        , "Unloads previously loaded files."]
        }}
     , {"reload",
        {["r"], many_args, "[FILENAMES]",
         [ "Reloads given files preserving the included scope."
-        , "Cleans user variables and functions."]
+        , "Cleans user variables, functions, types and contracts."]
```

Comment:
Types? Really?

Reply:
Yes, it uses `clear_context` which clears `typesdefs` and `typescope`.